lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -237,14 +237,15 @@ static size_t setKernelArgsForBuffers(cl_kernel kernel, const Instruction &I,
 
 void OpenCLFunction::fillBuffer(cl_mem buffer, uint64_t start, uint64_t len,
                                 float value, ElemKind elemKind,
-                                runtime::OpenCLDeviceBindings *devBindings) {
+                                runtime::OpenCLDeviceBindings *devBindings,
+                                std::vector<KernelLaunch> &kernelLaunches) {
   auto kernel =
       createKernel(getKernelName("splat", elemKind), devBindings->program);
   setKernelArg(kernel, 0, buffer);
   setKernelArg<cl_uint>(kernel, 1, start);
   setKernelArg(kernel, 2, value);
   enqueueKernel("splat", devBindings->commandQueue, kernel,
-                devBindings->deviceId, {(size_t)len}, kernelLaunches_);
```

Comment:
Is `kernelLaunches_` no longer used? Should it be removed from `OpenCLFunction`?